### PR TITLE
Protected update

### PR DESCRIPTION
**Pull request by TsunamiCode5**

### CODE REVIEW

File: app/controllers/destinations_controller.rb [hunk 1]
```diff
@@ -3,10 +3,16 @@ def index
         destinations = Destination.all
         render json: destinations
     end
+
+    def update
+        destination = Destination.find(params[:id])
+        destination.update(destination_params)
+        if destination.valid?
+            render json: destination
+        else
```
Comment (elysemontano) [inline app/controllers/destinations_controller.rb:12]: I don't see anything after the else here

**Review by elysemontano — CHANGES_REQUESTED** [review]
Double check the destinations_controller, it looks like you may be missing the second half of your conditional.

File: app/controllers/destinations_controller.rb (hunk 1 above)
Comment (TsunamiCode5) [reply app/controllers/destinations_controller.rb:12]: We have runaway code!

Comment (elysemontano) [reply app/controllers/destinations_controller.rb:12]: 🤣🏃‍♀️

**Review by elysemontano — APPROVED** [review]

**Pull request merged**